packages/core-mobile/app/store/rpc/handlers/avalanche_sendTransaction/avalanche_sendTransaction.ts

```diff
@@ -1,40 +1,6 @@
-import Config from 'react-native-config'
-import { AppListenerEffectAPI } from 'store'
-import * as Navigation from 'utils/Navigation'
-import AppNavigation from 'navigation/AppNavigation'
-import {
-  avaxSerial,
-  EVM,
-  EVMUnsignedTx,
-  UnsignedTx,
-  utils,
-  VM
-} from '@avalabs/avalanchejs'
-import { rpcErrors } from '@metamask/rpc-errors'
-import { selectActiveAccount } from 'store/account/slice'
-import networkService from 'services/network/NetworkService'
-import { selectIsDeveloperMode } from 'store/settings/advanced/slice'
-import walletService from 'services/wallet/WalletService'
+import { VM } from '@avalabs/avalanchejs'
 import { RpcMethod, RpcRequest } from 'store/rpc/types'
-import * as Sentry from '@sentry/react-native'
-import Logger from 'utils/Logger'
 import { Avalanche } from '@avalabs/core-wallets-sdk'
-import { getAddressByVM } from 'store/account/utils'
-import { getProvidedUtxos } from 'utils/getProvidedUtxos'
-import {
-  showTransactionErrorToast,
-  showTransactionSuccessToast
-} from 'utils/toast'
-import {
-  ApproveResponse,
-  DEFERRED_RESULT,
-  HandleResponse,
-  RpcRequestHandler
-} from '../types'
-import { parseRequestParams } from './utils'
-
-const GLACIER_URL = Config.GLACIER_URL
-const GLACIER_API_KEY = Config.GLACIER_API_KEY
 
 export type AvalancheTxParams = {
```

Comment:
same here. you should move these types to the avalanche module.

Reply:
moved.